### ISSUE TITLE
Support multiple AnsibleSequences in file

### ISSUE THREAD

**Issue by mauriciofierrom** [mauriciofierrom/ansible-vault-nvim#3]
A file with content like:

```yaml
- some: "some"
   var: !vault |
          ANSIBLE...
- other: "other"
   var: !vault |
          ANSIBLE...
```

Returns a list of dicts with the variables (AnsibleSequences):

```python
[{'some': 'some', 'var': 'a'}, {'other': 'other', 'var': 'a'}]
```

**Comment by mauriciofierrom** [comment]
Closed via #5.

**Comment by mauriciofierrom** [comment]
Closed via #5
